canonical-websites/snapcraft.io
# Snap details screenshots

Fixes #344

Adds screenshots view to snap details page.

### QA

- ./run
- go to snap detail page of a snap with screenshots
- screenshots should be visible
- click on a screenshot - lightbox should be opened
- if there is more screenshots you should be able to scroll through them with buttons
- go to snap without screenshots - screenshots should not be rendered
- try to disable JavaScript
- go to snap with screenshots
- screenshots should still be displayed correctly (without buttons) and should be scrollable with mouse scroll

<img width="1065" alt="screen shot 2018-03-21 at 18 03 45" src="https://user-images.githubusercontent.com/83575/37726682-37ab753c-2d36-11e8-9401-832d881f8dad.png">
<img width="1042" alt="screen shot 2018-03-21 at 18 32 48" src="https://user-images.githubusercontent.com/83575/37726751-62b25d2c-2d36-11e8-9917-c65faed7bf65.png">
